packages/components/ng-ovh-payment-method/src/components/choice/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+import component from './component';
+
+const moduleName = 'ngOvhPaymentMethodChoice';
```

Comment:
Not sure about the `ng` prefix

Reply:
Hmm all modules inside `ng-ovh-payment-method`are prefixed so, but if it is not okay, what do you suggest ? Removing it, and for all the inner modules ?

Reply:
If it is done so, we can keep the `ng` prefix then :) 
I got confused between the `ovhManager` modules (specific to manager app) and components